### DISCUSSION TITLE
Image upload failed using Minio

### DISCUSSION THREAD

**Discussion by allonair** [outline/outline#6190] [Self Hosting]
Hi,

I am running both Outline and Minio on Ubuntu server using Slack authentication.
I can successfully login to Outline using Slack and publish documents with no problem.

However, whenever I upload an image it fails and get below ERR_FAILED message due to 'no 'Access-Control-Allow-Origin' header is present on the requested resource.'

![Screenshot_2023-11-21_09 30 45PM](https://github.com/outline/outline/assets/126546824/a6b188dd-014e-426d-a72c-523c1572be77)

I have added Cors Allow Origin on Minio config page as below.

![Screenshot_2023-11-21_09 32 27PM](https://github.com/outline/outline/assets/126546824/feaeb10e-e263-48e1-aad6-f5d3e9a46b02)

Below is my docker.env file config for Minio.

AWS_ACCESS_KEY_ID=S58WFwKYXVykH6qWL87e
AWS_SECRET_ACCESS_KEY=leU6pLE9EAq4d1tWqfLz286Siu64UNISZvCRAHOv
AWS_REGION=ap-northeast-1
AWS_S3_UPLOAD_BUCKET_URL=http://s3.cmdrover.com:9000
AWS_S3_UPLOAD_BUCKET_NAME=outline-assets
AWS_S3_FORCE_PATH_STYLE=false
AWS_S3_ACL=private

What else can I check...?
I have searched tons of documents but failed to solve this.....
I really want to use Outline...!
Please please help!!

Thanks,
Sonia

**Comment by tommoor** [comment]
Please leave some details on how you resolved the issue for future community members.
